File: Lista1/server7.py
import socket

HOST = 'localhost'
PORT =  10000

s = socket.socket(socket.AF_INET,socket.SOCK_STREAM)
s.bind((HOST,PORT))
s.listen()

print('Aguardando conexão ')
conn, ender = s.accept()

print('Conectado em: ', ender)

while True:
    data = conn.recv(2048)	
	
    if not data:
        print('Fechando conexão')
        conn.close()
        break	
	
    data = data.decode()

    idade,anos = data.split('_')
    idade = int(idade)
    anos = int(anos)

    if idade >= 65:
        msg = 'Pode se aposentar'
    elif anos >= 30:
        msg = 'Pode se aposentar'
    elif idade >= 60 and anos >= 25:
        msg = 'Pode se aposentar'
    else:
        msg = 'Não pode se aposentar'
    
    data = str.encode(msg)
    conn.sendall(data)

File: Lista2/server7.java
import java.io.*;
import java.net.ServerSocket;
import java.net.Socket;
import java.util.Scanner;

public class server7 extends Thread {
	private Socket concurrentSocket;

	public server7(Socket clientSocket) {
		this.concurrentSocket = clientSocket;
	}

	public static void main(String[] args) {
		try {
			try (ServerSocket serverSocket = new ServerSocket(10000)) {
				System.out.println("Aguardando Cliente");
				while (true){
					Socket clientSocket = serverSocket.accept();
					System.out.println("Cliente conectado:"+clientSocket);
					server7 client = new server7(clientSocket);
					client.start();
				}
			}
		} catch (IOException i){
	}
}
    // dados
    public void run(){
		try {
			InputStream inputStream = concurrentSocket.getInputStream();
            
			try (Scanner scanner = new Scanner(inputStream)) {
				OutputStream outputStream = concurrentSocket.getOutputStream();
				PrintWriter out = new PrintWriter(outputStream, true);
        
                String data = scanner.next();
                String[] dados = data.split("_");
                String msg;
                float age;

                int idade = Integer.parseInt(dados[0]);
                int anos = Integer.parseInt(dados[1]);
            
                if (idade >= 65)
                    msg = "Pode se aposentar";
                else if (anos >= 30)
                    msg = "Pode se aposentar";
                else if (idade >= 60 && anos >= 25)
                    msg = "Pode se aposentar";
                else
                    msg = "Nao pode se aposentar";
            
                out.println(msg);
                System.out.println("Dados enviados ao cliente.");
			} catch (NumberFormatException e) {
			
				e.printStackTrace();
			}
		} 
        catch (IOException i){}
	}
}